src/cls/otp/cls_otp_client.h

```diff
@@ -28,11 +28,12 @@ namespace rados {
         static int get(librados::ObjectReadOperation *op,
                        librados::IoCtx& ioctx, const std::string& oid,
                        const std::list<std::string> *ids, bool get_all, std::list<otp_info_t> *result);
+#endif
```

Comment:
see comment in lines 25-26. these functions should not be called from inside the RGW (they are using IoCtx - hence sync calls).
they should be inside the `ifndef`